test/crash_test/src/crash_mgmt.c

```diff
@@ -42,16 +43,25 @@ struct mgmt_group crash_test_nmgr_group = {
     .mg_group_id = MGMT_GROUP_ID_CRASH
 };
 
+static char how_str[64];
+static struct os_callout mynewt_os_mgmt_crash_callout;
+
+static void
+crash_cb(struct os_event *unused)
+{
+    crash_device(how_str);
+}
+
 static int
-crash_test_nmgr_write(struct mgmt_cbuf *cb)
+crash_test_nmgr_write(struct mgmt_ctxt *cb)
 {
-    char tmp_str[64];
+    int delay_ms = 200;
```

Comment:
We should make this a syscfg with a default of 100ms, a small value is probably better for regression suites